### PR TITLE
feat(autodevops)!: use .base_deploy_app_chart_stage

### PR DESCRIPTION
**Pull request by douglasduteil**

### CODE REVIEW

File: autodevops_simple_app.yml [hunk 1]
```diff
@@ -119,7 +108,7 @@ Deploy app (dev):
   variables:
     HOST: ${CI_ENVIRONMENT_SLUG}-${CI_PROJECT_NAME}.${KUBE_INGRESS_BASE_DOMAIN}
   environment:
-    name: review/${CI_COMMIT_REF_NAME}-dev
+    name: ${CI_COMMIT_REF_NAME}-dev
```
Comment (revolunet) [inline autodevops_simple_app.yml:111]: would be good to keep review prefix to keep folded environments ?

Comment (douglasduteil) [reply autodevops_simple_app.yml:111]: Alright
i just that one can not see pods details on in the folders
image

Comment (revolunet) [reply autodevops_simple_app.yml:111]: i think it works in gitlab.com ?

Comment (douglasduteil) [reply autodevops_simple_app.yml:111]: > i think it works in gitlab.com ?

dunno

Comment (revolunet) [reply autodevops_simple_app.yml]: en effet je trouve pas de preuve que ca marche :/

**Pull request merged**